nuxeo-features/nuxeo-platform-collections/nuxeo-platform-collections-core/src/main/resources/OSGI-INF/collection-versioning-contrib.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<component name="org.nuxeo.ecm.collections.core.versioning.policies" version="1.0">
+
+  <require>org.nuxeo.ecm.platform.el.service</require>
+
+  <extension target="org.nuxeo.ecm.core.versioning.VersioningService" point="policies">
+    <policy id="no-versioning-for-collection-before-update" beforeUpdate="true" increment="NONE" order="2">
```

Comment:
Worth adding a comment here that the `order=2` has a special meaning or is in a "reserved" zone, and adding the same kind of comment in all other places where we use small `order` values.